### ISSUE TITLE
Can't add a hook

### ISSUE THREAD

**Issue by thany** [typicode/husky#1043]
**Troubleshoot**
- [x] Before creating an issue, please check:
https://typicode.github.io/husky/#/?id=troubleshoot

If you're migrating from husky 4, see:
https://typicode.github.io/husky/#/?id=migrate-from-v4-to-v7

**Context**
I'm using Husky in the command prompt.

Here's what I do:
```
npx husky add .husky/pre-push "npm run prepush"
```

Also tried with single-quotes, as per the documentation. Both don't work.

Output:

```
Usage:
  husky install [dir] (default: .husky)
  husky uninstall
  husky set|add <file> [cmd]
```

Errrr, that's what I'm doing, aren't I? 🤨
What does it want me to do instead?! No, I'm calling this a bug. I'm doing *exactly* as the usage instructs me to do, and I can literally copypaste from the documentation and it still won't work:

```
> npx husky add .husky/commit-msg 'npx --no-install commitlint --edit "$1"'
Usage:
  husky install [dir] (default: .husky)
  husky uninstall
  husky set|add <file> [cmd]
```

Sorry mates, but this is not okay. How did this get past testing?

It used to be so simple in Husky v4... Just add a thing in the package.json, and done. Worked everytime, everywhere. I wonder why you had to make everything difficult and unreliable... This is not the first problem I've run into that's new in v7 😑

Please give your huskies, and Husky, some more love 😘

**Comment by yiidot** [comment]
Got the same issue, emm.
You can try this temporarily:
```
npx husky add .husky/pre-push "npm-run-prepush"
```
and then change "npm-run-prepush" to "npm run prepush".
It works in my project.

**Comment by typicode** [comment]
If you're on Windows, it's an issue with npx.
You can do `node node_modules/.bin/husky add ...` instead.

**Comment by thany** [comment]
@yiidot 
> `npx husky add .husky/pre-push "npm-run-prepush"`

Huh, weird. That works. Thanks.

@typicode 
> If you're on Windows, it's an issue with npx.

Are you sure? I've executed other npx commands just fine. Can you workaround it?

**Comment by typicode** [comment]
Pretty sure. As a quick test, you can add `console.log('argv =', process.argv)` at line 2 of `node_modules/husky/lib/bin.js` and run `npx husky add .husky/commit-msg 'npx --no-install commitlint --edit "$1"'`.

If I remember correctly, you can then run the same command with `node node_modules/.bin/husky add ...` and you should get a different output.

No workaround possible at husky level.

There's probably a tricky parsing issue related to Windows and rn I don't have the time to create a proper issue on npm repo.

Duplicate https://github.com/typicode/husky/issues/1010

I've updated docs BTW.

**Comment by luisenricke** [comment]
> If I remember correctly, you can then run the same command with node node_modules/.bin/husky add ... and you should get a different output.

This solution works in WSL too, thanks.

**Comment by thany** [comment]
Beep boop, bad bot!

We only got a workaround. I haven't seen a solution yet.

**Comment by melvingaye** [comment]
It's definitely a windows npx issue. Use yarn as a workaround

**Comment by thany** [comment]
I'm not going to install yarn for this. First off, our project doesn't use yarn. Secondly, Husky should just work with whatever NPM versions come with the Node.js versions that are officially supported.

The real fix is in Husky to properly find its directories, or fix whatever else is going wrong. Not being able to "see" the `.husky` directory, for example, is entirely fixable by looking for it in the right place.

Or fix it by requiring a very specific NPM version (not recommended as one is not always in control of this, for example when using NVM).

**Comment by typicode** [comment]
Seems to be fixed in npm 8.1.1
